floris/tools/__init__.py

```diff
@@ -19,7 +19,7 @@
     '__name__', '__package__', '__path__', '__spec__', 'cut_plane',
     'floris_interface',
     'layout_functions', 'optimization', 'plotting', 'power_rose',
-    'visualization']
+    'visualization', 'wind_rose']
```

Comment:
This file was removed #775 so it's no longer needed here. I removed this specific line in #819, so this might be a result of merge conflicts.

Reply:
Ah, gotcha, I misread the merge conflict current vs incoming. I was merging v4 in and it was listed in the incoming, or so I thought. I will remove it.

Reply:
@rafmudaf Yep, I got the two switched up, just went back to that commit and re-merged. Good catch!

Reply:
Got it, thanks for fixing that